pyproject.toml

```diff
@@ -98,13 +98,14 @@ filterwarnings = [
 ]
 junit_family="xunit2"
 markers = [
+    "base_data_context: mark test as being relevant to BaseDataContext, which will be removed during refactor",
+    "cloud: mark test as being relevant to Great Expectations Cloud.",
     "docs: mark a test as a docs test.",
-    "unit: mark a test as a unit test.",
-    "integration: mark test as an integration test.",
     "e2e: mark test as an E2E test.",
     "external_sqldialect: mark test as requiring install of an external sql dialect.",
-    "cloud: mark test as being relevant to Great Expectations Cloud.",
-    "base_data_context: mark test as being relevant to BaseDataContext, which will be removed during refactor",
+    "integration: mark test as an integration test.",
+    "slow: mark tests taking longer than our slow threshold.",
```

Comment:
Where can the slow threshold be found?

Reply:
it's set to 1.0 sec right now but not enforced anywhere in the OSS repo - I think we should add it here

Reply:
@cdkini Where is it set to 1 sec?

Reply:
Is it this which is 2 secs?
https://github.com/great-expectations/great_expectations/blob/bbd0795358c0817065ad3e3fe5ed6f920b1d83a8/tasks.py#L210